src/status_im/browser/core.cljs

```diff
@@ -90,8 +90,10 @@
 
 (defn resolve-ens-contenthash-callback [hex]
   (let [proto-code (subs hex 2 4)
-        hash (when hex (multihash/base58 (multihash/create :sha2-256 (subs hex 8))))]
-    (if (and (#{"00" "01"} proto-code) hash (not= hash resolver/default-hash))
+        hash (when hex (multihash/base58 (multihash/create :sha2-256 (subs hex 12))))]
+    ;; We only support IPFS
+    ;; TODO Once more implementations / providers are published this will have to be improved
+    (if (and (= "e3" proto-code) hash (not= hash resolver/default-hash))
       (re-frame/dispatch [:browser.callback/resolve-ens-multihash-success proto-code hash])
```

Comment:
sorry for confusion, we support both . ipfs and swarm, so changes needed in resolve-ens-multihash-success, also in resolve-ens-content-callback code needs to be changed

Reply:
How do we support swarm? Do we have a way to access files on swarm?

Reply:
yes, there is swarm-gateways.net/bzz:/